Functions.Templates/Documentation/rabbitMQTrigger.md

```diff
@@ -0,0 +1,26 @@
+#### Settings for RabbitMQ trigger
+
+The settings specify the following properties.
+
+- `queueName` : The name of the queue to listen to.
+- `connectionStringSetting` : The name of an app setting that has connection string.
+- `type` : Must be set to *rabbitMqTrigger*.
```

Comment:
delete `type` and `direction` - does not apply to c# sample. Attribute take care of it.